client/src/App.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import store from './store';
-import Landing from './components/layout/Landing';
+import configureStore, { history } from './store';
+import { ConnectedRouter } from 'connected-react-router';
+import { Switch, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Landing from './components/landing/Landing';
+import Layout from './components/layout/Layout';
+import PrivateRoute from './components/layout/PrivateRoute';
 import './App.css';
```

Comment:
I need a better system for sorting imports. I've been spoiled by go doing it (mostly) automatically.

Reply:
Okay, well it's kinda hard because of multiple imports on one line, but my current plan is to at least separate by local repo vs vendor...